src/ape_test/providers.py

```diff
@@ -32,7 +32,17 @@ def chain_id(self) -> int:
 
     @property
     def gas_price(self) -> int:
-        # NOTE: Test chain doesn't care about gas prices
+        """Returns 0 because test chains do not care about gas prices."""
```

Comment:
Fun fact: test chains actually do care about gas now....

But, I have a branch (that relies on this one) that makes eth-tester work all over.

Reply:
rekt by EIP-1559

I think I was talking with steve or you at one point about setting the base fee in genesis to 0, but yeah I guess that would still affect this target gas calculation.... :thinking: 

if target gas were high enough (maybe 50m gas), and base fee were zero, then it should be impossible to have the algorithm trigger an upward adjustment, which only happens if the gas used in a block exceeds the target.

Reply:
In a test env, `base_fee + 1` should suffice, right? because everything just gets mined.